omod/src/main/java/org/openmrs/module/coreapps/page/controller/summarydashboard/ProgramEnrollmentsPageController.java

```diff
@@ -0,0 +1,78 @@
+package org.openmrs.module.coreapps.page.controller.summarydashboard;
+
+import org.apache.commons.lang.StringUtils;
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.node.ObjectNode;
+import org.openmrs.PatientProgram;
+import org.openmrs.Program;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.appframework.context.AppContextModel;
+import org.openmrs.module.appframework.domain.AppDescriptor;
+import org.openmrs.module.appframework.domain.Extension;
+import org.openmrs.module.appframework.service.AppFrameworkService;
+import org.openmrs.module.appui.UiSessionContext;
+import org.openmrs.module.coreapps.CoreAppsConstants;
+import org.openmrs.module.coreapps.CoreAppsProperties;
+import org.openmrs.module.emrapi.EmrApiProperties;
+import org.openmrs.ui.framework.annotation.SpringBean;
+import org.openmrs.ui.framework.page.PageModel;
+import org.openmrs.ui.framework.page.Redirect;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+public class ProgramEnrollmentsPageController {
+
+    public Object controller(@RequestParam("app") AppDescriptor app,
+                             @SpringBean EmrApiProperties emrApiProperties,
+                             @SpringBean("coreAppsProperties") CoreAppsProperties coreAppsProperties,
+                             @SpringBean("appFrameworkService") AppFrameworkService appFrameworkService,
+                             PageModel model,
+                             UiSessionContext sessionContext) {
+
+        if (!Context.hasPrivilege(CoreAppsConstants.PRIVILEGE_SUMMARY_DASHBOARD)) {
+            return new Redirect("coreapps", "noAccess", "");
+        }
+
+        model.addAttribute("app", app);
+
+        AppContextModel contextModel = sessionContext.generateAppContextModel();
+        model.addAttribute("appContextModel", contextModel);
+
+        List<Extension> includeFragments = appFrameworkService.getExtensionsForCurrentUser(app.getId() + ".includeFragments", contextModel);
+        Collections.sort(includeFragments);
+        model.addAttribute("includeFragments", includeFragments);
```

Comment:
It looks like we aren't using these "includeFragments" in the page itself, we are only using the "firstColumnFragments"? Or am I missing them.  If we aren't using them, let's remove this block?

Reply:
So, yes, let's remove this block (47-49) since we aren't using "includeFragments" in the rendered model.